src/App.js

```diff
@@ -1,34 +1,31 @@
 import React, { useState } from 'react';
-import './App.css';
+import { Route, Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+// import Game from './Game';
+import './App.scss';
 
-function App() {
-
-  const [started, setStarted] = useState(false);
 
-  const startGame = () => {
-    setStarted(true);
-  }
+function App() {
 
-  if (started) {
-    return (
-      <div>Game STARTED!</div>
-    )
-  } else {
     return (
```

Comment:
So three things, you can declare this using:
```
const App = function() {
```

I think declaring it as `function App()` might place it in the Global context.

Also, you can move this an arrow function, more people do it that way these days because it passes the context into the function.

finally, you can just use implicit return with the arrow because you don't do anything in the function right now except return the jsx.

```
const App = () => (<div className="App">......);
```